servers/visual/visual_server_scene.cpp

```diff
@@ -2088,8 +2088,10 @@ void VisualServerScene::_prepare_scene(const Transform p_cam_transform, const Ca
 				float zn = p_cam_projection.get_z_near();
 				Plane p(cam_xf.origin + cam_xf.basis.get_axis(2) * -zn, -cam_xf.basis.get_axis(2)); //camera near plane
 
+				Vector2 vp_he = p_cam_projection.get_viewport_half_extents();
 				float vp_w, vp_h; //near plane size in screen coordinates
-				p_cam_projection.get_viewport_size(vp_w, vp_h);
+				vp_w = vp_he.x;
```

Comment:
@clayjohn , @reduz 
This is a possible bug in [setup shadow maps .. compute coverage] caused by the previous confusing function name. Should vp_w and vp_h be in fact set to double the half extents?

Reply:
I just tested it out manually as I wasnt sure either. But the current implementation is definitely correct. It seems these should be half extents. When I tried doubling, or doubling and subtracting 1.0, everything broke.

Reply:
> I just tested it out manually as I wasnt sure either. But the current implementation is definitely correct. It seems these should be half extents. When I tried doubling, or doubling and subtracting 1.0, everything broke.

Ok that's good to know. In that case I could possibly simplify it by removing the vp_w and vp_h variables, renaming the vp_he and using that later in the code. Which do you think would be clearer? Also I'm not sure about the comment, I'm not sure it is correct, see above.

Reply:
The comment definitely seems wrong. 

Renaming the variables to increase clarity is definitely helpful. :)